### PR TITLE
Fix golint issues in pkg/kubelet/{prober,secret}

### PR DESCRIPTION
**Pull request by obitech**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Improve code quality with removing golint issues in:
- `pkg/kubelet/prober/...`
- `pkg/kubelet/secret`

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm
/ok-to-test

thanks for this change :)

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Comment by obitech** [comment]
@mattjmcnaughton all tests are green now!

**Comment by mattjmcnaughton** [comment]
sweet!

/lgtm

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Review by fejta — APPROVED** [review]
/lgtm
/approve

**Comment by obitech** [comment]
@mattjmcnaughton @derekwaynecarr is this mergable or are any open issues with this PR?

**Comment by mattjmcnaughton** [comment]
/assign @vishh 

Someone from `/pkg/kubelet/OWNERS` (like @vishh ) will need to approve, and then you should be good to go.

**Review by fejta — COMMENTED** [review]
/approve

File: pkg/kubelet/prober/testing/fake_manager.go [hunk 1]
```diff
@@ -17,19 +17,29 @@ limitations under the License.
...
 
-// Unused methods.
-func (_ FakeManager) AddPod(_ *v1.Pod)        {}
-func (_ FakeManager) RemovePod(_ *v1.Pod)     {}
-func (_ FakeManager) CleanupPods(_ []*v1.Pod) {}
-func (_ FakeManager) Start()                  {}
+// Unused methods below.
 
-func (_ FakeManager) UpdatePodStatus(_ types.UID, podStatus *v1.PodStatus) {
+// AddPod simulates adding a Pod.
```
Comment (tallclair) [inline pkg/kubelet/prober/testing/fake_manager.go]: nit: I would just say `AddPod is unimplemented`. Same below.

**Review by tallclair — COMMENTED** [review]
/approve

Some of the comments golint insists on annoy me, but I'd prefer to have golint checks.

**Pull request merged**